src/Assets/Flysystem/ProtectedAssetAdapter.php

```diff
@@ -34,8 +34,8 @@ protected function findRoot($root) {
         }
 
         // Use environment defined path
-        if(defined('SS_PROTECTED_ASSETS_PATH')) {
-            return SS_PROTECTED_ASSETS_PATH;
+        if(getenv('SS_PROTECTED_ASSETS_PATH')) {
+            return getenv('SS_PROTECTED_ASSETS_PATH');
```

Comment:
Isn't it more efficient to avoid calling `getenv` twice and instead do something like below?
```
if($protetedAssetsPath=getenv('SS_PROTECTED_ASSETS_PATH')) {
   return $protectedAssetsPath;
}

If then there are a number of other places one reduce the calls to `getenv` :)